jx_sqlite/query_table.py

```diff
@@ -288,7 +288,12 @@ def query(self, query):
                     data = Data()
                     for c in index_to_columns.values():
                         if c.push_child == ".":
-                            data[c.push_name] = c.pull(result.data[0])
+                            if not data[c.push_name]:
```

Comment:
Using `not` is dangerous in Python:  Empty lists, empty objects and zero, are all falsey yet could be legitimate data. You may be tempted to write `if data[c.push_name] is None`, as per Python standards, but since we are using `wrap()`, `Data` and `Null` (instead of `None`); you should write it as  `if data[c.push_name] == None:`